packages/lesswrong/components/form-components/SubforumNotifications.tsx

```diff
@@ -0,0 +1,36 @@
+import startCase from 'lodash/startCase';
```

Comment:
Oh dear this got counted as a delete and create rather than a rename, not going to change though

Reply:
There are some changes, and also I renamed the file from SubforumEmailNotifications to SubforumNotifications, so git is counting it as deleting "SubforumEmailNotifications.tsx" and adding "SubforumNotifications.tsx"